runtime/Cargo.toml

```diff
@@ -9,7 +9,7 @@ repository = "https://github.com/cennznet/cennznet"
 [dependencies]
 # third-party dependencies
 codec = { version = "1.3.0", package = "parity-scale-codec", default-features = false, features = ["derive"] }
-cennznut = { git = "https://github.com/cennznet/cennznut-rs", default-features = false, branch = "0.4.0" }
+cennznut = { git = "https://github.com/cennznet/cennznut-rs", default-features = false, branch = "0.5.0" }
```

Comment:
Why do we have a branch instead of the version here? Should we fix cennznut versioning @holygits?